packages/blockchain-wallet-v4-frontend/src/components/Layout/index.ts

```diff
@@ -69,6 +69,10 @@ export const SceneSubHeaderText = styled(Text)`
   color: ${props => props.theme.grey600};
   font-weight: 500;
   margin-bottom: 30px;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
```

Comment:
This is a pretty global component, I have no problem with the change but have you checked all the spots look good?

Reply:
It's the subheader underneath empty states. Earn Interest is the only one with a 'Learn More' link that needed to be centered to the Text, so it didn't affect the others.